src/main/default/lwc/lookup/lookup.js

```diff
@@ -1,7 +1,7 @@
 import { LightningElement, api } from 'lwc';
 
-const MINIMAL_SEARCH_TERM_LENGTH = 2; // Min number of chars required to search
-const SEARCH_DELAY = 300; // Wait 300 ms after user stops typing then, peform search
+const MINIMAL_SEARCH_TERM_LENGTH = 0; // Min number of chars required to search
```

Comment:
A question here is that on some components it is good to start it from 0. Maybe this could be set over api and leave default on 0. And api woule overwrite it.